proxyclient/m1n1/hv/__init__.py

```diff
@@ -1564,8 +1522,10 @@ def load_hook(data, segname, size, fileoff, dest):
 
         print(f"Physical memory: 0x{phys_base:x} .. 0x{mem_top:x}")
         print(f"Guest region start: 0x{guest_base:x}")
-
-        self.entry = macho.entry - macho.vmin + guest_base
+        if(is_macho_bin == True):
```

Comment:
Style: no parentheses around the condition in Python

Reply:
resolved with the refactor